### ISSUE TITLE
Track player joins/departure

### ISSUE THREAD

**Issue by ProgrammerDan** [Civcraft/AFK-Player-GC#12]
TPS tanks *quickly* and in very precise intervals. We might be able to focus the plugin's targeting apparatus by tracking and prioritizing player joins on intervals that record significant TPS drop. Similarly, we could track TPS upticks and see who departed the server in those periods. 

We shouldn't exclusively deal with those players, but could be a strong indicator; e.g. skip the movement analyzer and directly scan their environments.

**Comment by Maxopoly** [comment]
I did some very interesting testing on civtest earlier, (which also resulted in the server crashing), could you come on mumble later?

**Comment by ProgrammerDan** [comment]
Sadly I can't tonight, but if typing isn't too terrible toss me an email?

**Comment by Maxopoly** [comment]
https://forum.civcraft.co/t/lag-i-wonder-if-there-isnt-something-cumulative-that-creates-lag-over-time/811/14?u=maxopoly

That's the short version of it.

**Comment by ProgrammerDan** [comment]
Yeah, very useful testing. Matches our rebalance from last night, and I can probably move chests to 0 as well.
